qsr_lib/cfg/defaults.ini

```diff
@@ -1,2 +1,3 @@
 [arg_relations_distance]
-relations_and_values = {"0": 5., "1": 15., "2": 120.}
+;relations_and_values = {"0": 5., "1": 15., "2": 120.}
+relations_and_values = {}
```

Comment:
```python
except ConfigParser.NoOptionError:
            raise
```

OK, let me change it... It will take less time to do that that discussing it here :)